_posts/2018-11-29-blog-build-chrome-ringcentral-widgets-extension-for-crms.markdown

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title:  "Build Chrome RingCentral widgets extension for CRMs"
+date:   2018-11-29 13:31:07 +0800
+categories: blog
+---
+
+## Why is it need?
+
+- First, and the important reason, I guess, is Our clients need it, so for big CRM, like salesforce, we even build special app for it. For not that big CRMs, still have clients need it too, we may not build a special app for it. But we have the powerful widgets, so we can build a chrome extension easily.
+- Second, our RingCentral widgets is userful and powerful for these CRMs, we could greatly improve their work proccess.
+
+## What will we do in these extensions?
```

Comment:
I think it makes more sense to title this section as "What will these extensions provide?"